recipes/xqilla/all/patches/001-add-yajl-license.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/src/yajl/LICENSE.yajl b/src/yajl/LICENSE.yajl
```

Comment:
Do not create a patch for it, instead, extract directly from source file:

https://docs.conan.io/en/latest/howtos/extract_licenses_from_headers.html

Use [this](https://github.com/conan-io/conan-center-index/blob/4c258ca99120eec4316c4078e98cc5d56c3ddeaf/recipes/panzi-portable-endian/all/conanfile.py#L21) recipe as reference

Reply:
Patch removed, now getting license from header file.